podvm/Dockerfile.podvm

```diff
@@ -11,10 +11,17 @@ FROM ${BUILDER_IMG} AS podvm_builder
 ARG CLOUD_PROVIDER=libvirt
 ARG PODVM_DISTRO=ubuntu
 ARG AA_KBC=offline_fs_kbc
+# If not provided, uses system architecture
+ARG ARCH
 
 ENV CLOUD_PROVIDER ${CLOUD_PROVIDER}
 ENV PODVM_DISTRO ${PODVM_DISTRO}
 ENV AA_KBC ${AA_KBC}
+ENV ARCH ${ARCH}
+
+# Installs add-ons for foreign target, if required
+COPY hack/cross-build-extras.sh cross-build-extras.sh
```

Comment:
The podvm_builder should include all files under podvm which includes `hack/*`. This statement is not needed. Also we should avoid copying from local system unless absolutely needed to keep the container build process isolated from host.